flight/mcu/src/mcu_ac/main.cpp

```diff
@@ -156,12 +156,14 @@ static THD_FUNCTION(servo_THD, arg) {
     struct pointers *pointer_struct = (struct pointers *)arg;
     bool active_control = false;
 
+    ServoControl control(pointer_struct);
```

Comment:
`control` could be confused with the "active control" class we've yet to make. Lets call this instance something like `servo_control`